ui/app/utils/parse-pki-cert-oids.js

```diff
@@ -75,3 +76,14 @@ export const SIGNATURE_ALGORITHM_OIDs = {
   '1.2.840.10045.4.3.4': '512', // ECDSA-SHA512
   '1.3.101.112': '0', // Ed25519
 };
+
+// returns array of strings that correspond to model attributes
+// can be passed to display views in details pages containing certificates
+export const parsedParameters = [
```

Comment:
added so we could consistently display any model that has a certificate attribute.

this way if we update the parser at all and add OIDs, these attributes will be rendered by any relevant display components